lib/utils/cookie/cookieV1.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const pxutil = require('../pxutil');
+const Cookie = require('./cookie');
+
+class CookieV1 extends Cookie {
+
+    constructor(ctx, config) {
+        super();
+        this._pxCookie = ctx.cookies['_px'];
+        this._pxConfig = config;
+        this._pxContext = ctx;
+        this._cookieSecret = config.COOKIE_SECRET_KEY;
+    }
+
+    get score() {
+        return this.decodedCokie.s.b;
+    }
+
+    get hmac() {
+        return this.decodedCokie.h;
+    }
+
+    isCookieFormatValid(cookie) {
+        return pxutil.verifyDefined(cookie.t, cookie.s, cookie.s.a, cookie.s.b, cookie.u, cookie.v, cookie.h);
+    }
+
+    blockAction() {
+        // v1 cookie will always return captcha action
+        return 'c';
+    }
+
+    isSecure() {
+        const baseHmacStr = '' + this.time + this.decodedCokie.s.a + this.score + this.uuid + this.vid;
```

Comment:
It's there to force concatenation of the numbers. Otherwise the numbers are added before being concatenated to the uuid and vid.